src/components/Navigation/TopBar/index.ts

```diff
@@ -2,12 +2,12 @@ import ApplicationDrawer from './ApplicationDrawer/ApplicationDrawer';
 import { GuidelineSections } from './Guidelines/Guidelines';
 import { Notifications } from './Notifications/Notifications';
 import { Resources } from './Resources/Resources';
-import { Account } from './Account';
 import { Actions } from './Actions';
 import FieldSelector from './FieldSelector';
 import Guidelines from './Guidelines';
 import { ISettingsProps, Settings } from './Settings';
 import { TopBar as BaseTopBar } from './TopBar';
+import { Account } from 'src/components/Navigation/TopBar/Account/Account';
```

Comment:
![image](https://github.com/equinor/amplify-components/assets/17411650/00ed2f7b-797f-4095-919e-d4ed6d9a875e)

Reply:
(Edited comment, so changed the order XD)

Reply:
Same with this one